Contents/Code/siteUltrafilms.py

```diff
@@ -0,0 +1,147 @@
+            maxscore = score
+
+        Log("MaxScore:" + str(maxscore))
+
+    if maxscore < 100:
+        Log("no exact match, trying with normal search")
+        url = PAsearchSites.getSearchSearchURL(siteNum) + encodedTitle
+        headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.47 Safari/537.36'}
+
+        try:
+            searchResults = HTML.ElementFromURL(url)
+        except:
+            request = urllib.Request(url, headers=headers)
+            response = urllib.urlopen(request, context=ssl.SSLContext(ssl.PROTOCOL_TLSv1))
+            htmlstring = response.read()
+            searchResults = fromstring(htmlstring)
+
+        for searchResult in searchResults.xpath('//ul[@class="listing-videos listing-tube"]/*[div[@class="format-infos"]/div[@class="time-infos"]]'):
+            titleNoFormatting = searchResult.xpath('.//a')[0].get('title').strip()
+            Log("titleNoFormatting: " + titleNoFormatting)
+            actors = searchResult.xpath('.//img')[0].get('alt')
+            Log(actors)
+            sceneUrl = searchResult.xpath('.//a')[0].get('href')
+            curID = sceneUrl.replace('/','_').replace('?','!')
+            Log("curID: " + curID)
+            score = 100 - Util.LevenshteinDistance(searchTitle.lower(), titleNoFormatting.lower())
+            results.Append(MetadataSearchResult(id = curID + "|" + str(siteNum), name = titleNoFormatting + " with "+ actors + " [" + PAsearchSites.getSearchSiteName(siteNum) + "] ", score = score, lang = lang))
+
+    return results
+        
+
+def update(metadata,siteID,movieGenres,movieActors):
+    pageURL = str(metadata.id).split("|")[0].replace('_', '/').replace('?','!')
+    Log('scene url: ' + pageURL)
+    headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.47 Safari/537.36'}
+    try:
+        detailsPageElements = HTML.ElementFromURL(pageURL)
+    except:
+        request = urllib.Request(pageURL, headers=headers)
+        response = urllib.urlopen(request, context=ssl.SSLContext(ssl.PROTOCOL_TLSv1))
+        htmlstring = response.read()
+        detailsPageElements = fromstring(htmlstring)
+
+    # Summary
+    siteName = PAsearchSites.getSearchSiteName(siteID)
+    metadata.studio = siteName
+    try:
+        #site dont give a description
+        metadata.summary = 'No description'
+        metadata.title = detailsPageElements.xpath('//span[@property="name"]')[3].text_content().strip()
+    except:
+        pass
+
+    # Collections / Tagline
+    metadata.collections.clear()
+    metadata.tagline = siteName
+    metadata.collections.add(siteName)
+
+    # Genres
+    try:
+        genres = detailsPageElements.xpath('//div[@itemprop="keywords"]//ul//li//a')
+        if len(genres) > 0:
+            for genreLink in genres:
+                genreName = genreLink.text_content().strip().lower()
```

Comment:
Could you update this line to `genreName = genreLink.text_content().replace('Movies','').strip().lower()`

It'll remove the term "Movies" from all genres.

Reply:
done